ui/src/app/shared/genericComponents/shared/converter.ts

```diff
@@ -127,4 +120,21 @@ export namespace Converter {
   export const HIDE_VALUE: Converter = (ignore): string => {
     return '';
   };
+
+  /**
+   * Calculates the otherPower: the power, that can't be assigned to a consumer
+   * 
+   * @param evcss the evcss
+   * @param consumptionMeters the "CONSUMPTION_METERED" meters 
+   * @param currentData the currentData
+   * @returns the otherPower
+   */
+  export const CALCULATE_CONSUMPTION_OTHER_POWER = (evcss: EdgeConfig.Component[], consumptionMeters: EdgeConfig.Component[], currentData: CurrentData): number => {
+    let activePowerTotal = currentData.allComponents['_sum/ConsumptionActivePower'] ?? null;
+    let evcsChargePowerTotal = evcss?.map(evcs => currentData.allComponents[evcs.id + '/ChargePower'])?.reduce((prev, curr) => Utils.addSafely(prev, curr), 0) ?? null;
+    let consumptionMeterActivePowerTotal = consumptionMeters?.map(meter => currentData.allComponents[meter.id + '/ActivePower'])?.reduce((prev, curr) => Utils.addSafely(prev, curr), 0) ?? null;
+
```

Comment:
const should be used for the above three variables.

Reply:
ok